### PR TITLE
Feature/style old messages

### LINKED ISSUE #38: Stored messages displayed with a disabled state

### PR DESCRIPTION
**Pull request by luciagirasoles**
- wrap text to avoid horizontal scroll
- change name from extras to helpers
- rename file "extras.js" to "helpers.js"
- create function removeChildsByParentId(id) into helpers.js

close #38

### CODE REVIEW

File: public/assets/js/helpers.js [hunk 1]
```diff
@@ -10,3 +10,12 @@ document.getElementById("js-input-user-message").onkeypress = function(e) {
...
 };
+//remove childs from html element
+function removeChildsByParentId(id) {
+  var e = document.getElementById(id);
+  var child = e.lastElementChild;
+  while (child) {
+    e.removeChild(child);
+    child = e.lastElementChild;
+  }
+}
```
Comment (cristianbgp) [inline public/assets/js/helpers.js]: Well done, We are going to use it from switching channels

File: public/assets/js/helpers.js [hunk 2]
```diff
@@ -10,3 +10,12 @@ document.getElementById("js-input-user-message").onkeypress = function(e) {
   var chr = String.fromCharCode(e.which);
   if ("></".indexOf(chr) >= 0) return false;
 };
+//remove childs from html element
+function removeChildsByParentId(id) {
+  var e = document.getElementById(id);
```
Comment (yummta) [inline public/assets/js/helpers.js]: Remember that the names of the variables should be more descriptive

Comment (cdsandoval) [reply public/assets/js/helpers.js]: I agree with this one. I'm working on it as well.

File: public/assets/js/helpers.js [hunk 3]
```diff
@@ -10,3 +10,12 @@ document.getElementById("js-input-user-message").onkeypress = function(e) {
...
+  var child = e.lastElementChild;
```
Comment (cdsandoval) [inline public/assets/js/helpers.js]: Plese don't forget to use let or const.

**Pull request merged**